recipes/master.rb

```diff
@@ -84,6 +80,16 @@
   end
 end
 
+ruby_block 'delay salt-master service start' do
+  block do
+  end
+  notifies :start, 'service[salt-master]'
```

Comment:
> some issue with kitchen test which got resolved

Can you add more info here about that? (for future reference and full context on the change)